packs/spells/infectious-melody.json

```diff
@@ -19,7 +19,7 @@
+            "value": "<p>You call forth the chords of a tune so catchy that foes hearing it can't help but sing along, making it harder for them to communicate. Each target must attempt a Will save.</p><hr /><p><strong>Critical Success</strong> The creature is unaffected.</p>\n<p><strong>Success</strong> For 1 round, the creature can communicate only by singing along to the melody. It can speak only on its turn, and it must take the @UUID[Compendium.pf2e.actionspf2e.Item.Perform] action when it does. The result of the Performance check determines how well it sang, but it stills communicate the information regardless of the result, which has no further mechanical impact than the time spent to take the action. A single Perform check is sufficient for any amount of speaking on the creature's turn. This effect might interfere with verbal components of spells or other linguistic abilities that rely on precise speech; in order to use such an ability successfully, the creature must succeed at a @Check[flat|dc:5|showDC:all].</p>\n<p>In addition, the creature takes a -2 status penalty on linguistic Intimidation checks, as singing its threats saps away some of the menace. The creature's constant humming of the tune also makes it nearly impossible for the creature to remain undetected.</p>\n<p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Infectious Melody (Success)]</p>\n<p><strong>Failure</strong> As success, but the duration is 3 rounds.</p>\n<p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Infectious Melody (Failure)]</p>\n<p><strong>Critical Failure</strong> As success, but the duration is 1 hour.</p>\n<p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Infectious Melody (Critical Failure)]</p>"
```

Comment:
```suggestion
            "value": "<p>You call forth the chords of a tune so catchy that foes hearing it can't help but sing along, making it harder for them to communicate. Each target must attempt a Will save.</p><hr /><p><strong>Critical Success</strong> The creature is unaffected.</p>\n<p><strong>Success</strong> For 1 round, the creature can communicate only by singing along to the melody. It can speak only on its turn, and it must take the @UUID[Compendium.pf2e.actionspf2e.Item.Perform] action when it does. The result of the Performance check determines how well it sang, but it stills communicate the information regardless of the result, which has no further mechanical impact than the time spent to take the action. A single Perform check is sufficient for any amount of speaking on the creature's turn. This effect might interfere with verbal components of spells or other linguistic abilities that rely on precise speech; in order to use such an ability successfully, the creature must succeed at a @Check[flat|dc:5|showDC:all].</p>\n<p>In addition, the creature takes a -2 status penalty on linguistic Intimidation checks, as singing its threats saps away some of the menace. The creature's constant humming of the tune also makes it nearly impossible for the creature to remain @UUID[Compendium.pf2e.conditionitems.Item.Undetected].</p>\n<p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Infectious Melody (Success)]</p>\n<p><strong>Failure</strong> As success, but the duration is 3 rounds.</p>\n<p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Infectious Melody (Failure)]</p>\n<p><strong>Critical Failure</strong> As success, but the duration is 1 hour.</p>\n<p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Infectious Melody (Critical Failure)]</p>"
```